charts/radar-rest-sources-backend/templates/_helpers.tpl

```diff
@@ -83,22 +95,38 @@ Get the key for the secret object.
 */}}
 {{- define "radar-rest-sources-backend.secretKey" -}}
 {{- if eq .type "url" -}}
-    {{- .values.postgres.urlSecret.key | default "databaseUrl" -}}
+    {{- if .values.postgres.urlSecret }}
```

Comment:
@pvannierop we are running into issues because we are using RDS. I've added this to check if the `urlSecret` is present here before getting the key. I think in other parts it will still default to `host`, `user`, etc?